tests/phpunit/admin/class-wc-stripe-admin-notices-test.php

```
@@ -1023,4 +1023,133 @@ public function test_hide_notices_dismisses_ece_location_notice() {
+
+		$notices = new WC_Stripe_Admin_Notices();
+		$notices->subscription_check_detachment();
+
+		remove_filter( 'pre_http_request', $test_request );
+
+		$theorder = $original_order;
+		unset( $_GET['page'], $_GET['id'] );
+		WC_Stripe_Database_Cache::delete( 'payment_method_for_source_src_456' );
+
+		$this->assertCount( 0, $notices->notices );
+	}
```

PHPMD (2.15.0)

[error] test_subscription_check_detachment_not_shown_when_dismissed accesses the super-global variable $_GET. (undefined)

(Superglobals)